<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapView</class>
 <widget class="QWidget" name="MapView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>391</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QScrollBar" name="horizontalScrollBar">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGraphicsView" name="MapView"/>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="mapToolPanel">
       <item>
        <widget class="QToolButton" name="eventButton">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="main.qrc">
           <normaloff>:/main/res/icon/event.xpm</normaloff>:/main/res/icon/event.xpm</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">layerButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="panoramaButton">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="main.qrc">
           <normaloff>:/main/res/icon/panorama.xpm</normaloff>:/main/res/icon/panorama.xpm</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">layerButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="bottomLayerButton">
         <property name="text">
          <string>Down</string>
         </property>
         <property name="icon">
          <iconset resource="main.qrc">
           <normaloff>:/main/res/icon/layer_bottom.xpm</normaloff>:/main/res/icon/layer_bottom.xpm</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">layerButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="middleLayerButton">
         <property name="text">
          <string>Up</string>
         </property>
         <property name="icon">
          <iconset resource="main.qrc">
           <normaloff>:/main/res/icon/layer_middle.xpm</normaloff>:/main/res/icon/layer_middle.xpm</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">layerButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="topLayerButton">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="main.qrc">
           <normaloff>:/main/res/icon/layer_top.xpm</normaloff>:/main/res/icon/layer_top.xpm</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">layerButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="skyLayerButton">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="main.qrc">
           <normaloff>:/main/res/icon/sky.xpm</normaloff>:/main/res/icon/sky.xpm</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">layerButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="zoom1_1">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="main.qrc">
           <normaloff>:/main/res/icon/1_1.xpm</normaloff>:/main/res/icon/1_1.xpm</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">zoomButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="zoom1_2">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="main.qrc">
           <normaloff>:/main/res/icon/1_2.xpm</normaloff>:/main/res/icon/1_2.xpm</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">zoomButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="zoom1_4">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="main.qrc">
           <normaloff>:/main/res/icon/1_4.xpm</normaloff>:/main/res/icon/1_4.xpm</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">zoomButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="zoom1_8">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="main.qrc">
           <normaloff>:/main/res/icon/1_8.xpm</normaloff>:/main/res/icon/1_8.xpm</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">zoomButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="databaseButton">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="main.qrc">
           <normaloff>:/main/res/icon/tile_set.xpm</normaloff>:/main/res/icon/tile_set.xpm</iconset>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QScrollBar" name="verticalScrollBar">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="layerButtonGroup"/>
  <buttongroup name="zoomButtonGroup"/>
 </buttongroups>
</ui>
